README.md

```diff
@@ -187,6 +187,12 @@ Not sure how to start or have questions? MMM-GPT is an AI that answers questions
 
 **[Try MMM-GPT here.](https://mmm-gpt.com/)**
 
+## Explainer App: Streamlit App of MMM Concepts
```

Comment:
I suggest we put this higher up for visibility ... maybe after 
### Essential Reading for Marketing Mix Modeling (MMM) ?

Reply:
Done! 😁